dockerfiles/alpine/obam/Dockerfile

```diff
@@ -64,6 +64,7 @@ RUN \
     apk add --no-cache \
         bash \
         libxml2-utils \
+        unzip \
```

Comment:
@nalakagunathilaka this should come to the bottom. The Dockerfile best practice is to maintain the package list in the alphabetical order of the starting letter.

Reply:
Resolved with latest commit.